util/base1024/init.go

```diff
@@ -1,36 +1,36 @@
 package base1024
 
 import (
-	"golang.org/x/xerrors"
+	"fmt"
+	"html"
 	"io/ioutil"
 	"os"
 	"strconv"
 	"strings"
 )
 
-var Emojis []uint8
+const TAIL = "\\ud83c\\udfad"
 
-func Init() error {
-	//get from json file
-	f, err := os.Open("./emoji.json")
+var Emojis []string
+
+func init() {
+	f, err := os.Open("./emojis.json")
```

Comment:
Don't read this file in runtime.
If file isn't large, give it a `const string` var.